core/iwasm/libraries/libc-uvwasi/libc_uvwasi.cmake

```diff
@@ -8,21 +8,33 @@ set (LIBUV_VERSION v1.44.2)
 add_definitions (-DWASM_ENABLE_LIBC_WASI=1 -DWASM_ENABLE_UVWASI=1)
 
 include(FetchContent)
+include(FindPackageHandleStandardArgs)
+
+# Based on https://github.com/xenoscopic/libuv-cmake/blob/fc164590c6c723fd474e35b0b52022ebaecef745/Findlibuv.cmake
+find_path (libuv_INCLUDE_DIR NAMES uv.h)
+find_library (libuv_LIBRARY NAMES uv libuv)
+find_package_handle_standard_args(libuv libuv_LIBRARY libuv_INCLUDE_DIR)
```

Comment:
IIUC, L14 and L15 will search *uv.h* and *libuv.so* in the default path which usually is also the system path and doesn't include a directory of WAMR source tree. So I am not sure it will locate the libuv downloaded by CMake.

Reply:
This is not designed to load libuv downloaded by cmake. It is only downloaded below if there is no system installed libuv

Reply:
I've updated the original description for clarity

Reply:
Got it. It would be helpful if using system providing libuv.so. 

BTW, how about the version of system installed libuv? Should  be worried about the compatibility problem?

Reply:
And how to install the system packages of libuv and uvwasi? Could you help update the document?

Reply:
> And how to install the system packages of libuv and uvwasi? Could you help update the document?

@wenyongh this PR is not completed. Is this a good way to proceed? If so, I can make similar changes for uvwasi.

Reply:
It is good to me if it really resolves the issue that developers met, could you make the changes for uvwasi, and update the document? e.g. update the `Note` of `Configure LIBC` here:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/doc/build_wamr.md#configure-libc

Reply:
I've been working on this, but when the uvwasi source isn't available, the cmake build process errors that it cannot find any of the symbols. Either I'm incorrectly linking uvwasi or maybe it doesn't support linking.